src/widgets/scrollbar.rs

```diff
@@ -705,29 +708,34 @@ mod tests {
         );
     }
 
-    #[test]
-    fn test_renders_empty_with_content_length_is_zero() {
-        let mut buffer = Buffer::empty(Rect::new(0, 0, 2, 8));
-        let mut state = ScrollbarState::default().position(0);
+    #[rstest]
+    #[case("        ", 0)]
+    #[case("████████", 1)]
+    #[case("████████", 2)]
+    #[case("████████", 3)]
+    #[case("████████", 4)]
+    #[case("████████", 5)]
+    #[case("████████", 6)]
+    #[case("████████", 7)]
+    #[case("████████", 8)]
```

Comment:
It's not obvious why this is rendered like this, so it's worth adding a longer comment for this test that documents what's going on here.

Additionally, it would more intuitive to read these tests as input (content_length) -> output (expected) rather than the reverse.

I don't quite understand the meaning of the name of this test. " test_renders_empty_with_content_length_is_zero"

Edit:
So, this is a test of the vertical rendering. It might be useful to use a vertical looking element for the bars, and have this represented as an array e.g. `[" #", " #", ...]`.
That said, the tests all have the same data except the first one. So it's could be better to split this into two tests rather than going for the visual approach.